config/index.ts

```diff
@@ -81,13 +81,32 @@ export function writeConfig(options: WriteConfigOptions = {}): void {
   }
 }
 
+export function getHiddenOrDeprecatedConfigPath(
+  useHiddenConfig = false
+): string | null {
+  if (useHiddenConfig || CLIConfiguration.isActive()) {
+    return getConfigFilePath();
+  }
+  return config_DEPRECATED.getConfigPath();
+}
+
 export function getConfigPath(path?: string): string | null {
   if (CLIConfiguration.isActive()) {
     return getConfigFilePath();
   }
   return config_DEPRECATED.getConfigPath(path);
 }
 
+export function otherConfigFileExists(useHiddenConfig?: boolean): boolean {
```

Comment:
Could both of these utils be simplified into a single export of `configFileExists`. For example:
```js
export function configFileExists(usehiddenConfig?: boolean) {
    return useHiddenConfig
    ? configFile.configFileExists()
    : Boolean(config_DEPRECATED.getConfigPath())
}

// Then used in CLI like...

// Checking if the other exists:
const doesOtherConfigFileExist = configFileExists(!useHiddenConfig)

// Checking if both exist
const bothExist = configFileExists(true) && configFileExists(false)
```

Then the logic in our export doesn't need to be inverted (i.e. "other"), and also we'd only need a single export

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/172/commits/b40fdfb4fe2448ef929b7d50b11bcb1fca8d55ab